code/modules/research/xenoarchaeology/artifact/effect.dm

```diff
@@ -94,10 +94,10 @@ proc/GetAnomalySusceptibility(var/mob/living/carbon/human/H)
 		if(rig.suit_is_deployed() && !rig.offline)
 			protected += 1
 
-	if(istype(H.wear_suit,/obj/item/clothing/suit/space/anomaly))
+	if(istype(H.wear_suit,/obj/item/clothing/suit/space/void/hazardsuit/moebius))
```

Comment:
Why only the moebius subtype, shouldn't the base hev suit give extra protection too?